Structure_oM/MaterialFragments/LaminatedVeneerLumberParallel.cs

```diff
@@ -0,0 +1,202 @@
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Geometry;
+using BH.oM.Quantities.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace BH.oM.Structure.MaterialFragments
+{
+
+    [Description("Structural timber material of type Laminated Veneer Lumber without crossband veneers. To be used on structural elements and properties, or as a fragment of the physical material.\n" +
+                 "Generally only applicable for use in beam elements.\n" +
+                 "Note: Properties for LVL are not part of a harmonised standard and therefore vary between manufacturers and products.")]
+    public class LaminatedVeneerLumberParallel : BHoMObject, ITimber
+    {
+        /***************************************************/
+        /**** Properties - General and analysis         ****/
+        /***************************************************/
+
+        [Description("Name. A unique name is required for some structural packages to create and identify the object.")]
+        public override string Name { get; set; }
+
+        [Density]
+        [Description("Mean Density. Used to calculate mass. Called ρmean in most manufacturer documentation.")]
+        public virtual double Density { get; set; }
+
+        [Density]
+        [Description("Characteristic Density. Used to calculate other mechanical properties (not mass). Called ρk in most manufacturer documentation.")]
+        public virtual double DensityCharacteristic { get; set; }
+
+        [Ratio]
+        [Description("Dynamic Damping Ratio. Ratio between actual damping and critical damping.")]
+        public virtual double DampingRatio { get; set; }
+
+        [YoungsModulus]
+        [Description("Modulus Of Elasticity of the material to be used for Analysis. Ratio between stress and strain in all directions.\n" +
+                     "Values can be automatically populated based on material parameters by calling the SetAnalysisParameters method.\n" +
+                     "Vector defines stiffnesses as follows:\n" +
+                     "X - Stiffness along the local x-axis of the element. For most cases this will be the Parallel stiffness (E_0).\n" +
+                     "Y - Stiffness along the local y-axis of the element. For most cases this will be parallel to the transverse grain direction (E_90_edge) for Flatwise and perpendicular to the glue-planes (E_90_flat) for Egewise. For most beam/slab element cases this this will be the horizontal Perpendicular stiffness.\n" +
+                     "Z - Stiffness along the local z-axis of the element. For most cases this will be perpendicular to the glue-planes (E_90_flat) for Flatwise and parallel to the transverse grain direction (E_90_edge) for Egewise. For most beam/slab element cases this this will be the vertical Perpendicular stiffness.")]
```

Comment:
```suggestion
        [Description("Modulus Of Elasticity of the material to be used for Analysis. Ratio between stress and strain in all directions.\n" +
                     "Values can be automatically populated based on material parameters by calling the SetAnalysisParameters method.\n" +
                     "Vector defines stiffnesses as follows:\n" +
                     "X - Stiffness along the local x-axis of the element. For most cases this will be the parallel stiffness (E_0).\n" +
                     "Y - Stiffness along the local y-axis of the element. For most cases this will be parallel to the transverse grain direction (E_90_edge) for Flatwise and perpendicular to the glue-planes (E_90_flat) for Edgewise. For most beam/slab element cases this this will be the horizontal perpendicular stiffness.\n" +
                     "Z - Stiffness along the local z-axis of the element. For most cases this will be perpendicular to the glue-planes (E_90_flat) for Flatwise and parallel to the transverse grain direction (E_90_edge) for Edgewise. For most beam/slab element cases this this will be the vertical perpendicular stiffness.")]
```

Similar to LVLCrossbands.